<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="9" parallel="methods" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <parameter name="success" value="S Thread"/>
  <parameter name="Fail" value="F Thread"/>
  <listeners>
    <listener class-name="testng.sample.ListenerTest"/>
    <listener class-name="testng.sample.Retry_Transform"/>
  </listeners>
  <test thread-count="9" parallel="methods" name="Test(failed)">
    <groups>
      <run>
        <include name="group_of_groups"/>
      </run>
    </groups>
    <classes>
      <class name="testng.sample.TestcasesTest">
        <methods>
          <include name="testtofailTest2" invocation-numbers="0 0 0 0"/>
          <include name="testtofailTest1" invocation-numbers="0 0 0 0"/>
        </methods>
      </class> <!-- testng.sample.TestcasesTest -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
